frozendict/frozendict.pyi

```diff
@@ -1,6 +1,57 @@
-try:
-    from ._frozendict import *
-except ImportError:
-    from .core import *
+import collections
+from _typeshed import Self
+from collections.abc import Iterable, Iterator, Mapping
+from typing import Any, Generic, NoReturn, Tuple, TypeVar, Union, overload
 
-FrozenOrderedDict = frozendict
+_KT = TypeVar("_KT")
```

Comment:
I think _KT should be an Hashable

Reply:
It seems that restricting it to `Hashable` causes a lot of troubles... https://github.com/python/mypy/issues/9698